CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 This is the changelog of Grafeas server releases. For more information on
 versionining, see [versioning](docs/versioning.md) document.
 
+v0.1.5:
+  * Upgraded to golang 1.14.0
+  * Addeded last_scan_time to discovery occurrences
+  * Added support for Windows updates
+  * Changed resource type API Group to top level domain
```

Comment:
I'm not sure I follow: could you please point me to the PR for this? I can't remember what this is about.:)

Reply:
I realized that I got this from a commit description, not the PR. I think it makes sense to remove that line since it is part of the "Added API config annotations" https://github.com/grafeas/grafeas/pull/404/commits.

Reply:
Gotcha. I think we can omit this altogether in the release notes, as those changes concern the internal mechanism for client lib generation, which doesn't affect the OSS version or the community as a whole.